test/framework/clusterctl/client.go

```diff
@@ -82,6 +85,61 @@ func Init(ctx context.Context, input InitInput) {
 	Expect(err).ToNot(HaveOccurred(), "failed to run clusterctl init")
 }
 
+// InitWithBinary uses clusterctl binary to run init with the list of providers defined in the local repository.
+func InitWithBinary(_ context.Context, binary string, input InitInput) {
```

Comment:
I'm not sure what the idea behind the context param here (and the other funcs in this file is). Maybe just hand it over in case we need it later on?

I imagine at least in the funcs with binary we will probably never use it?